adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/IngMapper.java

```diff
@@ -0,0 +1,102 @@
+package de.adorsys.xs2a.adapter.service.ing;
+
+import de.adorsys.xs2a.adapter.service.ing.internal.api.model.*;
+import de.adorsys.xs2a.adapter.service.model.TokenResponse;
+import de.adorsys.xs2a.adapter.service.psd2.model.*;
+import de.adorsys.xs2a.adapter.service.psd2.model.Balance;
+import de.adorsys.xs2a.adapter.service.psd2.model.HrefType;
+import de.adorsys.xs2a.adapter.service.psd2.model.TransactionsResponse;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Mapper
+public interface IngMapper {
+    AccountList map(AccountsResponse value);
+
+    ReadAccountBalanceResponse map(BalancesResponse value);
+
+    @Mapping(target = "balances", ignore = true)
+    @Mapping(target = "links", ignore = true)
```

Comment:
I'm not 100% sure, but it seems to me, that one of the `@Mapping` will be ignored without `@Mappings` wrapping, as we use 1.2.0 MapStruct version (https://mapstruct.org/documentation/1.2/reference/html/#basic-mappings), not the 1.3.0 (https://mapstruct.org/documentation/dev/reference/html/#basic-mappings)

Reply:
From the first link:
> When using Java 8 or later, you can omit the @Mappings wrapper annotation and directly specify several @Mapping annotations on one method.

Reply:
Got it, thanks!